components/test_util/src/logging.rs

```diff
@@ -98,24 +97,7 @@ pub fn init_log_for_test() {
     .unwrap();
     let writer = output.map(|f| Mutex::new(File::create(f).unwrap()));
     // we don't mind set it multiple times.
-    let drain = CaseTraceLogger { f: writer };
-
-    // Collects following targets.
-    const ENABLED_TARGETS: &[&str] = &[
-        "tikv::",
-        "tests::",
-        "benches::",
-        "integrations::",
-        "failpoints::",
-        "raft::",
-        // Collects logs for test components.
-        "test_",
-    ];
-    let filtered = drain.filter(|record| {
```

Comment:
Could you add a blacklist please? Otherwise, test logs will be too noisy

Reply:
I suggest to compile a TiKV without any filtering, switch to debug level, and observe what we need to filter away.

Reply:
like `raft` and `grpcio`? But once they are added by default, it is hard to enable them except change the code. Maybe set them in Makefile manually is better?

Reply:
I mean third-party crates, like tokio and mio, they output too many logs in debug level.